libraries/default.rb

```diff
@@ -0,0 +1,68 @@
+def collect_github_repositories(github_token, org)
+  if ::ObjectSpace.const_defined?('Chef')
+    # Chef exists! Lets make sure Octokit is installed.
+    chef_gem 'octokit'
+  end
```

Comment:
Same here this is being installed already on line 92 of recipes/cookbook_uploader.rb